pages/system/inputs.js

```diff
@@ -127,21 +159,33 @@ _handleChange = (e) => {
         <System.Input
           label="Copy and paste (read only)"
           readOnly
-          name="fifteen"
+          name="exampleFour"
           copyable
-          value={this.state.fifteen}
+          value={this.state.exampleFour}
           onChange={this._handleChange}
         />
         <br />
         <System.CodeBlock>
-          {`<System.Input
-  label="Copy and paste (read only)"
-  readOnly
-  name="fifteen"
-  copyable
-  value={this.state.fifteen}
-  onChange={this._handleChange}
-/>`}
+          {`class ExampleCopyPaste extends React.Component {
+   state = { exampleFour: 'aaaaa-bbbbb-ccccc-ddddd-eeee' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`